modules/KalturaSupport/components/streamSelector.js

```diff
@@ -21,8 +21,6 @@
 				"nextStream": 221,   // Add ] Sign for next stream
 				"prevStream": 219,   // Add [ Sigh for previous stream
 				"defaultStream": 220, // Add \ Sigh for default stream
-				"openMenu": 83, // Add S Sigh for open menu
-				"closeMenu": "shift+83" // Add Shift+S Sigh for close menu
```

Comment:
Yes we can remove them since the new interface of the stream selector (Now part of the dual screen new design) they do not preform any action. 

#3446 I closed.